scripts/imageSize.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+
+let fileArray = [];
+
+const read = dir =>
+  fs
+    .readdirSync(dir)
+    .reduce(
+      (files, file) =>
+        fs.statSync(path.join(dir, file)).isDirectory()
+          ? files.concat(read(path.join(dir, file)))
+          : files.concat(path.join(dir, file)),
+      []
+    )
+    .filter(file => typeof file === 'string')
+    .map(image => {
+      if (fs.lstatSync(image).size > 1000000) {
+        fileArray.push(image);
+      }
+    });
+
+read('src/images/');
+
+if (fileArray.length > 0) {
```

Comment:
This can be just `if (fileArray.length) {}` since `0` will evaluate to false, and that's the only case we care about.